src/inventory_ui.cpp

```diff
@@ -560,11 +560,8 @@ nc_color inventory_entry::get_invlet_color() const
 {
     if( !is_selectable() ) {
         return c_dark_gray;
-    } else if( get_player_character().inv->assigned_invlet.count( get_invlet() ) ) {
-        return c_yellow;
-    } else {
-        return c_white;
     }
+    return c_yellow;
```

Comment:
I think yellow and white are used to distinguish user-assigned and auto-assigned inventory letters.

Reply:
I wasn't sure about this change because I don't play with the auto-assigned inventory letters option (sidenote: it assigns the keys I use for navigation and it assigns digits preventing me from dropping/inserting X items at a time making the feature less than useful. Although both of these problems are bugs that can be fixed separately IMO).

Is it useful to the player for us to distinguish between user-assigned and auto-assigned inventory letters/keybindings?

NB: Thanks for the help @Qrox, it's much appreciated!

Reply:
I haven't used it in a while but the last time I used it user-assigned ones behave differently. I don't remember the details but I think user-assigned ones persist even if you drop the item and pick it back up, while auto-assigned ones can change after dropping and picking up.

Reply:
ahhhhhh might as well keep the color scheme as-is then, I'm already making enough changes here :)